src/Lykke.ClientGenerator/Infrastructure/AopProxy.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+
+namespace Lykke.ClientGenerator.Infrastructure
+{
+    /// <summary>
+    /// Helper to generate a calls wrapping proxy
+    /// </summary>
+    public class AopProxy : DispatchProxy
```

Comment:
Looks like it could be internal

Reply:
DispatchProxy requires its inheritors to be accessible via reflexion.